### PR TITLE
Addresses wording issues from 16.10 assessment

### PR DESCRIPTION
**Pull request by dominoFire**
## Bug

Fixes: https://github.com/NuGet/Client.Engineering/issues/972

Regression? Last working version:

## Description

This PR applies guidance received from wording check evaluation. Words are rephrased to reflect more precise meaning.

## PR Checklist

- [x] PR has a meaningful title
- [x] PR has a linked issue.
- [x] Described changes

- **Tests**
  - ~[ ] Automated tests added~
  - [x] Test exception: No product source code change. Test changes already covered.
  - ~[ ] N/A~

- **Documentation**
  - ~[ ] Documentation PR or issue filled~
  - [x] N/A

### CODE REVIEW

File: src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs [hunk 1]
```diff
@@ -155,7 +155,7 @@ public static async Task<EnvDTE.ProjectItems> GetProjectItemsAsync(EnvDTE.Projec
             }
             if (createIfNotExists)
             {
-                // The JS Metro project system has a bug whereby calling AddFolder() to an existing folder that
+                // The JS Modern UI (Widnows 8 era) project system has a bug whereby calling AddFolder() to an existing folder that
```
Comment (erdembayar) [inline src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs]: Widnows => Windows

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs [hunk 2]
```diff
@@ -51,7 +51,7 @@ public sealed class UIActionEngine
         /// <summary>
         /// Perform an install or uninstall user action.
         /// </summary>
-        /// <remarks>This needs to be called from a background thread. It may hang on the UI thread.</remarks>
+        /// <remarks>This needs to be called from a background thread. It may make the UI thread stop responding.</remarks>
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs:54]: use freeze like in other places where you've replaced hang?

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs [hunk 3]
```diff
@@ -195,7 +195,7 @@ private static string GenerateUpgradeReport(string projectName, string backupPat
         /// <summary>
         /// Perform the multi-package update action.
         /// </summary>
-        /// <remarks>This needs to be called from a background thread. It may hang on the UI thread.</remarks>
+        /// <remarks>This needs to be called from a background thread. It may make the UI thread stop responding.</remarks>
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs:198]: freeze?

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs (hunk 2 above)
Comment (jebriede) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs:54]: @nkolev92 the guidance is to replace "hang" with "stop responding" when referring to UI or threads. The freeze concept that he used in other place was when it didn't make sense to use "stop responding" such as in the test code where the parameter is "-hang" and "-stopresponding" just doesn't make a lot of sense.

File: src/NuGet.Clients/NuGet.VisualStudio.Implementation/VsTemplateWizard.cs [hunk 4]
```diff
@@ -463,7 +463,7 @@ void IWizard.RunStarted(object automationObject, Dictionary<string, string> repl
             // VsPackageInstaller and VsPackageUninstaller. Because, no powershell scripts get executed
             // as part of the operations performed below. Powershell scripts need to be executed on the
             // pipeline execution thread and they might try to access DTE. Doing that under
-            // ThreadHelper.JoinableTaskFactory.Run will consistently result in a hang
+            // ThreadHelper.JoinableTaskFactory.Run will consistently result in the UI stop responding
```
Comment (jebriede) [inline src/NuGet.Clients/NuGet.VisualStudio.Implementation/VsTemplateWizard.cs]: replace "result in" with "make":
ThreadHelper.JoinableTaskFactory.Run will consistently make the UI stop responding.

File: src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs [hunk 5]
```diff
@@ -260,7 +260,7 @@ private string EvaluatePrompt()
                     // on it. Note that a default prompt function as defined in Profile.ps1 will simply return
                     // a string "PM>". This will always work. However, a custom "prompt" function might call
                     // Write-Host and NuGet will explicity switch to the main thread using JTF.
-                    // If the main thread was blocked then, it will consistently result in a hang.
+                    // If the main thread was blocked then, it will consistently result in the UI stop responding
```
Comment (jebriede) [inline src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs]: replace "result in" with "make"

File: src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs (hunk 1 above)
Comment (jebriede) [reply src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs]: Can you make this JS Windows Store app like you did below?

File: src/NuGet.Clients/NuGet.VisualStudio.Common/ServiceLocator.cs [hunk 6]
```diff
@@ -62,7 +62,7 @@ public static void InitializePackageServiceProvider(IAsyncServiceProvider provid
 
             // then try to find the service as a component model, then try dte then lastly try global service
             // Per bug #2072, avoid calling GetGlobalService() from within the Initialize() method of NuGetPackage class.
-            // Doing so is illegal and may cause VS to hang. As a result of that, we defer calling GetGlobalService to the last option.
+            // Doing so is illegal and may cause VS to stop responding. As a result of that, we defer calling GetGlobalService to the last option.
```
Comment (jebriede) [inline src/NuGet.Clients/NuGet.VisualStudio.Common/ServiceLocator.cs:65]: "cause VS to stop responding" -> "make VS stop responding"

**Review by jebriede — APPROVED** [review]

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs (hunk 3 above)
Comment (dominoFire) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs:198]: I will follow with current guidance to keep consistency.

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs (hunk 2 above)
Comment (dominoFire) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs:54]: I will follow with current guidance to keep consistency.

Comment (nkolev92) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs:54]: Can you link that guidance? 

I followed https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1322667 but the links there don't take me to any particula recommendations, I'm sure I'm missing something.

**Review by zivkan — APPROVED** [review]

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs (hunk 2 above)
Comment (nkolev92) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs:54]: I found it through [an e-mail](https://policheck.azurewebsites.net/Pages/DisplayTermDetails.aspx), anyone has any idea how to get to that through the issues?

**Review by erdembayar — APPROVED** [review]

**Review by erdembayar — APPROVED** [review]

**Pull request merged**